### ISSUE TITLE
Failure to read input file

### ISSUE THREAD

**Issue by Madu86** [merzlab/QUICK#127]
The following input file is not properly read by QUICK.

<img width="669" alt="Screen Shot 2021-03-06 at 4 47 05 PM" src="https://user-images.githubusercontent.com/46697377/110221894-c6c92100-7e9c-11eb-839d-c18f3d43d06c.png">

As you can see, there is a tab between atom labels and x coordinate column. If you print out coordinates from here: https://github.com/merzlab/QUICK/blob/master/src/modules/quick_molspec_module.f90#L376-L377 this is what you get. 

```
        atom. direct.   coordinates (A)                     coordinates (au)
           1           1   0.0000000000000000        0.0000000000000000     
           1           2   2.4000000000000000E-002   4.5353427189255439E-002
           1           3  -6.3200000000000006E-002 -0.11943069159837266     
           2           1   0.0000000000000000        0.0000000000000000     
           2           2   1.4976000000000000        2.8300538566095392     
           2           3  -1.7299999999999999E-002  -3.2692262098921628E-002
           3           1   0.0000000000000000        0.0000000000000000     
           3           2   2.1631999999999998        4.0878555706582231     
           3           3   1.1657999999999999        2.2030427257180829     
           4           1   0.0000000000000000        0.0000000000000000     
           4           2   1.4784999999999999        2.7939600874714232     
           4           3   2.3913000000000002        4.5189021015694388     
           5           1   0.0000000000000000        0.0000000000000000     
           5           2   5.3900000000000003E-002  0.10185623856253617     
           5           3   2.4680000000000000        4.6638440959617675     
           6           1   0.0000000000000000        0.0000000000000000     
           6           2 -0.61909999999999998       -1.1699294488695016     
           6           3   1.2273000000000001        2.3192608828905499     
           7           1   0.0000000000000000        0.0000000000000000     
           7           2   2.1867000000000001        4.1322641347810363     
           7           3  -1.3364000000000000       -2.5254300039883737     
           8           1   0.0000000000000000        0.0000000000000000     
           8           2 -0.51040000000000008      -0.96451621822483247     
           8           3   3.5432999999999999        6.6958666066536994     
           9           1   0.0000000000000000        0.0000000000000000     
           9           2 -0.69780000000000009       -1.3186508955276020     
           9           3  -1.0350999999999999       -1.9560555201499290     
          10           1   0.0000000000000000        0.0000000000000000     
          10           2   1.9963000000000002        3.7724602790796098     
          10           3   3.2724000000000002        6.1839397972549790     
          11           1   0.0000000000000000        0.0000000000000000     
          11           2  -1.6497000000000002       -3.1174812014214459     
          11           3   1.2542000000000000        2.3700945158651736     
          12           1   0.0000000000000000        0.0000000000000000     
          12           2   3.2795000000000001        6.1973568527984666     
          12           3  -1.2538000000000000       -2.3693386254120195     
          13           1   0.0000000000000000        0.0000000000000000     
          13           2   1.8978999999999999        3.5865112276036619     
          13           3  -1.9314000000000000       -3.6498170530553313     
          14           1   0.0000000000000000        0.0000000000000000     
          14           2   1.8954000000000000        3.5817869122714479     
          14           3  -1.9325000000000001       -3.6518957518015056     
          15           1   0.0000000000000000        0.0000000000000000     
          15           2   3.2580000000000000        6.1567277409414256     
          15           3   1.2297000000000000        2.3237962256094753   
```  

The input reader must be capable of handling these things.  Lets fix this. We also have to try different experiments to break the reader and make sure its robust.

**Comment by Madu86** [comment]
@agoetz Please let us know your suggestions.

**Comment by agoetz** [comment]
String handling is quite limited in Fortran. A quick fix in `subroutine rdword` in the code block here https://github.com/merzlab/QUICK/blob/65030409c21e87604d8196236b1f7afb7c25da56/src/subs/rdword.f90#L29-L43
would be to use both tab and space characters as whitespace 
```
if ( (string(i:i) == ' ' ) .or. (string(i:i) == achar(9)) ) then
```
and also remove the label from the do loop, and format the file better.

**Comment by agoetz** [comment]
If you don't know what to do with your time, here is a discussion about string libraries for Fortran
https://github.com/fortran-lang/stdlib/issues/69

**Comment by Madu86** [comment]
@agoetz Thanks for the suggestions. @Altaimorin Can you take care of this issue?

**Comment by agoetz** [comment]
This has been fixed in PR #152 so I will close this issue.
